ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/dhcp-servers.yml

```diff
@@ -0,0 +1,34 @@
+dhcp_servers:
+  - vrf: TEST
```

Comment:
lets add more VRFs to test sorting with `default` (e.g. `defauls` and `defaulu`

Reply:
Im not sure what you mean sorting with "default".

Reply:
Sorted!